File: src/main/java/no/nav/fo/veilarbregistrering/arbeidssoker/resources/ArbeidssokerperiodeDto.java
package no.nav.fo.veilarbregistrering.arbeidssoker.resources;

public class ArbeidssokerperiodeDto {

    private String fraOgMedDato;
    private String tilOgMedDato;

    public ArbeidssokerperiodeDto(String fraOgMedDato, String tilOgMedDato) {
        this.fraOgMedDato = fraOgMedDato;
        this.tilOgMedDato = tilOgMedDato;
    }

    public String getFraOgMedDato() {
        return fraOgMedDato;
    }

    public String getTilOgMedDato() {
        return tilOgMedDato;
    }
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/arbeidssoker/ArbeidssokerperiodeDto.kt
package no.nav.fo.veilarbregistrering.arbeidssoker

import no.nav.fo.veilarbregistrering.bruker.Foedselsnummer
import java.time.LocalDateTime

data class ArbeidssokerperiodeDto(
    val id: Int,
    val foedselsnummer: Foedselsnummer,
    val fra: LocalDateTime,
    val til: LocalDateTime? = null
)
